src/main/app/document/DocumentManagementController.ts

```diff
@@ -224,133 +296,128 @@ export class DocumentManagerController extends PostController<AnyObject> {
+      console.log('File data... : ', fileData);
+
       const obj = {
         id: fileData[0]['originalname'],
         name: fileData[0]['originalname'],
       };
 
-      if (req.session.userCase['applicantUploadFiles']) {
+      console.log('ID details: ', obj.id);
+      console.log('name details: ', obj.name);
+
+      if (isApplicant === YesOrNo.YES) {
         req.session.userCase.applicantUploadFiles?.push(obj);
-        console.log('content in upload files: ', obj);
-        //return res.redirect(UPLOAD_DOCUMENT);
       } else {
-        req.session.userCase['applicantUploadFiles'] = [];
-        req.session.userCase.applicantUploadFiles?.push(obj);
+        req.session.userCase.respondentUploadFiles?.push(obj);
+
+        console.log('Respondent files []', respondentFiles);
       }
     }
 
-    //const documentManagementClient = this.getDocumentManagementClient(req.session.user);
-    //const filesCreated = await documentManagementClient.create({
-    //  files: req.files,
-    //  classification: Classification.Public,
-    //});
-
-    // const filesCreated = (req.files as Express.Multer.File[]).map(file => {
-    //   return {
-    //     originalDocumentName: file.originalname,
-    //   };
-    // });
-
-    console.log('3');
-    const newUploads: ListValue<Partial<UploadDocumentList> | null>[] = [];
-    // const newUploads: ListValue<Partial<UploadDocumentList> | null>[] = filesCreated.map(file => ({
-    //   id: generateUuid(),
-    //   value: {
-    //     id: "aaaaaaa",
-    //     value: {
-    //       parentDocumentType: 'Witness Statement',
-    //       DocumentType: 'Witness Statement',
-    //       partyName: 'Sonal Saha',
-    //       isApplicant: 'Yes',
-    //       uploadedBy: 'Uploaded by Sonali Saha',
-    //       dateCreated: '12/07/2022',
-    //       documentUploadedDate: '12/07/2022',
-    //       citizenDocument: {
-    //         document_url: 'abcd',
-    //         document_filename: file.originalDocumentName,
-    //         document_binary_url: 'abcd',
-    //       },
-    //     },
-    //   },
-    // }));
-    //  const documentsKey = 'applicant1DocumentsUploaded';
-    //  const updatedDocumentsUploaded = newUploads.concat(req.session.userCase?.[documentsKey] || []);
-    //  req.session.userCase = await req.locals.api.triggerEvent(
-    //    req.session.userCase?.id,
-    //    { [documentsKey]: updatedDocumentsUploaded },
-    //     CITIZEN_UPDATE
-    //  );
-    req.session.save(() => {
-      if (req.headers.accept?.includes('application/json')) {
-        res.json(newUploads.map(file => ({ id: file.id, name: getFilename(file.value) })));
-      } else {
-        const redirectUrl =
-          UPLOAD_DOCUMENT +
-          '?' +
-          'caption=' +
-          req.query.parentDocumentType +
-          '&document_type=' +
-          req.query.documentType;
-        res.redirect(redirectUrl);
-      }
-    });
+    let redirectUrl;
+
+    if (isApplicant === YesOrNo.YES) {
```

Comment:
change the condition to YES === isApplicant

Reply:
changed